README.md

```diff
@@ -1,92 +1,25 @@
-# Rust code formatting RFCs
+# Rust style team repository
 
-This repository exists to decide on a code style for Rust code, to be enforced
-by the Rustfmt tool. Accepted RFCs live in the `text` directory and form a
-specification for formatting tools.
+This is the home of the Rust style team. The style team is responsible for
```

Comment:
```suggestion
This is the home of the [Rust style team]. The style team is responsible for
```

Reply:
I tend to prefer inline links, but a link definitely makes sense here.

```suggestion
This is the home of the [Rust style team](https://www.rust-lang.org/governance/teams/lang#Style%20team). The style team is responsible for
```